src/Kunstmaan/MediaBundle/AdminList/MediaAdminListConfigurator.php

```diff
@@ -51,6 +54,10 @@ public function __construct(
 
         $this->folder = $folder;
         $this->request = $request;
+
+        // Thumbnail view should display 24 images, list view 250
+        $viewMode = $request->get('viewMode');
```

Comment:
When no view mode is in the request query, it will still display 250 items but the default viewMode is thumbnail.